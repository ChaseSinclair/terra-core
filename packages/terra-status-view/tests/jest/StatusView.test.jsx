import React from 'react';
import StatusView from '../../src/StatusView';

describe('StatusView', () => {
  const defaultRender = <StatusView type="custom" />;

  // Snapshot Tests
  it('should render a default component', () => {
    const wrapper = shallow(defaultRender);
    expect(wrapper).toMatchSnapshot();
  });
});
