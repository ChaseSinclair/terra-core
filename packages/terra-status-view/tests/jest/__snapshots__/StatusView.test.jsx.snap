// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusView should render a default component 1`] = `
<div
  className="status-view"
>
  <div
    className="icon"
  />
</div>
`;
